CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Optimizely Python SDK Changelog
 
+## 3.10.0
+September 16th, 2021
+
+### New Features
+* Added new public properties to OptimizelyConfig. 
+  - sdk_key and environment_key [#338] (https://github.com/optimizely/python-sdk/pull/338)
+  - attributes and events [#339] (https://github.com/optimizely/python-sdk/pull/339)
+  - experiment_rules, delivery_rules, audiences and audiences in OptimizelyExperiment 
+    - [#342] (https://github.com/optimizely/python-sdk/pull/342)
+    - [#351] (https://github.com/optimizely/python-sdk/pull/351/files)
+* For details please refer to our documentation page:
+  - Python-sdk: [https://docs.developers.optimizely.com/full-stack/docs/optimizelyconfig-python]
+
+* OptimizelyFeature.experiments_map is now deprecated. Please use OptimizelyFeature.experiment_rules and OptimizelyFeature.delivery_rules. [#360] (https://github.com/optimizely/python-sdk/pull/360)
```

Comment:
```suggestion
* OptimizelyFeature.experiments_map of OptimizelyConfig is now deprecated. Please use OptimizelyFeature.experiment_rules and OptimizelyFeature.delivery_rules. [#360] (https://github.com/optimizely/python-sdk/pull/360)
```

I believe most clients are not familiar with OptimizelyFeature and we do not want to scare them of this deprecation message. It'll be good to point out this is for OptimizelyConfig only.